Dockerfile

```diff
@@ -1,16 +1,18 @@
 # This file is part of REANA.
-# Copyright (C) 2017, 2018 CERN.
+# Copyright (C) 2017, 2018, 2020 CERN.
```

Comment:
I guess also 2019?

Would it be possible to keep a range on these cases? e.g. `Copyright (C) 2017-2020 CERN. cc/ @tiborsimko @diegodelemos

Reply:
Guess this is similar case to [this one](https://github.com/reanahub/reana-workflow-monitor/pull/39#discussion_r525906051), that's why I haven't added 2019

Reply:
I see.. anyway not a big deal! 

:shipit: